recipes/craq/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -x -e
+
+# Put craq in share
+CRAQ_DIR=${PREFIX}/share/CRAQ
+mkdir -p ${CRAQ_DIR}
+cp -r * ${CRAQ_DIR}
+chmod +x ${CRAQ_DIR}/bin/craq
```

Comment:
Could you please leave a comment on why you put the binary in `${PREFIX}/share/CRAQ/bin` and not in `${PREFIX}/bin` and later create a softlink. Looks to me, like just depositing your binary in `${PREFIX}/bin` would avoid the softlink all together